pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -214,13 +231,167 @@ The default value for `loadPowerFactorConstant` property is `false`.
 
 **plausibleActivePowerLimit**  
 The `plausibleActivePowerLimit` property is an optional property that defines a maximal active power limit for generators to be considered as participating elements for:
-- slack distribution (if `balanceType` equals to `PROPORTIONAL_TO_GENERATION_P_MAX` or `PROPORTIONAL_TO_GENERATION_P`)
+- slack distribution (if `balanceType` equals to any of the `PROPORTIONAL_TO_GENERATION_<any>` types)
 - slack selection (if `slackBusSelectionMode` equals to `LARGEST_GENERATOR`)
 
 The default value is $$5000 MW$$.
 
-**addRatioToLinesWithDifferentNominalVoltageAtBothEnds**  
-The `addRatioToLinesWithDifferentNominalVoltageAtBothEnds` property is an optional property used for lines that are connected to two voltage level with different nominal voltages. If this property equals `true`, a structural ratio is taken into account. The default value of this parameter is `true`. This property should normally be left to its default value `true` to get correct load-flow results, and will be removed in a future release.
+**slackBusPMaxMismatch**  
+When slack distribution is enabled (`distributedSlack` set to `true` in LoadFlowParameters), this is the threshold below which slack power
+is considered to be distributed.
+The default value is $$1 MW$$.
+
+**voltagePerReactivePowerControl**  
+Whether simulation of SVCs slope should be enabled (See `voltage per reactive power control` extension).  
```

Comment:
SVC: better use static var compensators

Reply:
I think that the voltage per reactive power control means that a static var compensator in `VOLTAGE` control with an extension `VoltagePerReactivePowerControl` will use its slope to modulate its voltage target at regulating terminal.